### DISCUSSION TITLE
multiple room simulation

### DISCUSSION THREAD

**Discussion by spetitbx** [Ifsttar/I-Simpa#338] [Q&A] [answered]
I'm trying to simulate a experimental lab with all the sources of noise in an other room. I followed the tutorial3 (industrial room) and i'm always stunk with the simulation limited to only one room (SPPS or CTR). Sound is not calculated in the other rooms. Punctual receivers give Inf numbers. How to extend the simulation to the other rooms?
![image](https://user-images.githubusercontent.com/115698519/195552568-65d993aa-c02a-4a2e-9efd-6afac4330cd7.png)

i'm using I-simpa1.3.4, build 2020-12-23T16:57:21 on win10 64 bits

**Comment by nicolas-f** [comment, marked as answer]
Hi,

You have to set a Transmission loss value in the intermediate walls.

https://i-simpa-wiki.readthedocs.io/fr/latest/tutorial_industrial_hall.html#define-surface-materials

**Reply by spetitbx** [reply]
thanks
